luigi/__init__.py

```diff
@@ -49,6 +49,7 @@
 
 from .tools import range  # just makes the tool classes available from command line
 
+__version__='2.7.1'
```

Comment:
use whitespaces around '='. Its the standard practice and the function you defined in setup.py is also not able to get the version number due to missing whitespaces.